examples/introduction/simplesystem.ipynb

```diff
@@ -1460,106 +564,83 @@
       "Calculating diversification multiplier for CORN\n",
+      "No mapping applied for CORN\n",
       "Calculating subsystem position for SP500\n",
       "Calculating volatility scalar for SP500\n",
       "Calculating instrument value vol for SP500\n",
       "Calculating instrument currency vol for SP500\n",
-      "Getting block value for SP500\n",
-      "Loading csv data for SP500\n",
-      "Loading csv instrument config\n",
-      "Loading csv data for SP500\n",
-      "Getting fx rates for SP500\n",
-      "Loading csv instrument config\n",
-      "Loading csv fx data\n",
-      "Loading csv fx data\n",
-      "Loading csv fx data\n",
       "Calculating combined forecast for SP500\n",
       "Calculating forecast weights for SP500\n",
-      "Calculating raw forecast weights for SP500\n",
       "Calculating capped forecast for SP500 ewmac32\n",
       "Calculating raw forecast SP500 for ewmac32\n",
-      "Loading csv data for SP500\n",
       "Calculating capped forecast for SP500 ewmac8\n",
       "Calculating raw forecast SP500 for ewmac8\n",
-      "Loading csv data for SP500\n",
       "Calculating diversification multiplier for SP500\n",
+      "No mapping applied for SP500\n",
       "Calculating subsystem position for US10\n",
       "Calculating volatility scalar for US10\n",
       "Calculating instrument value vol for US10\n",
       "Calculating instrument currency vol for US10\n",
-      "Getting block value for US10\n",
-      "Loading csv data for US10\n",
-      "Loading csv instrument config\n",
-      "Loading csv data for US10\n",
-      "Getting fx rates for US10\n",
-      "Loading csv instrument config\n",
-      "Loading csv fx data\n",
-      "Loading csv fx data\n",
-      "Loading csv fx data\n",
       "Calculating combined forecast for US10\n",
       "Calculating forecast weights for US10\n",
-      "Calculating raw forecast weights for US10\n",
       "Calculating capped forecast for US10 ewmac32\n",
       "Calculating raw forecast US10 for ewmac32\n",
-      "Loading csv data for US10\n",
       "Calculating capped forecast for US10 ewmac8\n",
       "Calculating raw forecast US10 for ewmac8\n",
-      "Loading csv data for US10\n",
       "Calculating diversification multiplier for US10\n",
-      "2016-11-07   -10.494962\n",
-      "2016-11-08   -15.857233\n",
-      "2016-11-09   -16.012381\n",
-      "2016-11-10   -23.799042\n",
-      "2016-11-11   -34.351904\n",
+      "No mapping applied for US10\n",
+      "index\n",
+      "2020-11-17     1.440444\n",
+      "2020-11-18     3.504983\n",
+      "2020-11-19     9.385244\n",
+      "2020-11-20    13.455780\n",
+      "2020-11-23    13.255764\n",
       "Freq: B, dtype: float64\n"
      ]
+    },
+    {
+     "data": {
+      "text/plain": "<Figure size 432x288 with 0 Axes>"
+     },
+     "metadata": {},
+     "output_type": "display_data"
     }
    ],
    "source": [
-    "run simplesystem.py"
+    "%run simplesystem.py"
```

Comment:
This is the only actual intended change, but the notebook seemed to do the rest.